src/rachanaGJun20/String/MaxDifferIndexInStrings.java

```diff
@@ -0,0 +1,31 @@
+/*Program 5: Find all differ index from given arrays.
+               i/p:- arr1: {"Maulik", "Harsh", "Krishna", "Technocredits"}
+                       arr2: {"Maulik","Harsh","Technocredits","Krishna"}		
+               o/p:- Values are not matching at index -> 2, [arr1 value is Krishna, arr2 value is Technocredits] 
+                        Values are not matching at index -> 3, [arr1 value is Technocredits, arr2 value is Krishna]*/
+package rachanaGJun20.String;
+
+public class MaxDifferIndexInStrings {
+
+	void findMaxDifferIndex(String[] arr1, String[] arr2) {
```

Comment:
Are you performing the same operation which type of class name and method name you have given here.

Reply:
here I am not getting you.... please let me know.